docs/framework/wpf/graphics-multimedia/how-to-control-the-fill-of-a-composite-shape.md

```diff
@@ -19,39 +19,39 @@ The <xref:System.Windows.Media.GeometryGroup.FillRule%2A> property of a <xref:Sy
   
  The following illustration shows the shape created in the previous example.  
   
- ![Screenshot: FillRule Property of EvenOdd](./media/fillruleevenoddfirstone.png "FillRuleEvenOddFirstOne")  
+ ![Diagram showing the FillRule property of EvenOdd.](./media/how-to-control-the-fill-of-a-composite-shape/fillrule-evenodd-property.png)  
   
  In the illustration above, notice that the center and 3rd ring are not filled. This is because a ray drawn from any point within either of those two rings passes through an even number of segments. See illustration below:  
```

Comment:
```suggestion
 In the previous illustration, notice that the center and third ring are not filled. This is because a ray drawn from any point within either of those two rings passes through an even number of segments. See the following illustration:  
```